src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -74,10 +74,10 @@ private void testCreateInstructorAction() throws Exception {
         InstructorAttributes instructor = new InstructorAttributes();
         
         String shortName = "Instrúctör";
-        instructor.name = "AHPUiT Instrúctör";
-        instructor.email = "AHPUiT.instr1!@gmail.tmt";
+        instructor.name = "AHPUiT Instrúctör WithNameInEmail";
```

Comment:
`WithNameInEmail` doesn't indicate that I should expect weird chars in email address of this guy. Furthermore, most test data follow the pattern that user name is partially/fully included in the email address, so there is no reason to mention it in just this test case. Either omit it or use a more meaningful 'name'?